### ISSUE TITLE
Issue with relational calculated field

### ISSUE THREAD

**Issue by LevelUpGit** [rezo-labs/directus-extension-computed-interface#44]
When using relational ASUM(a, b) we are getting "Failed to load resource: the server responded with a status of 400 ()", we can't figure it out, what is causing it. We are using 10.2.0 version of Directus.

Update: We have modified name of id column, and that is what the problem was. Your name of ID column must be ID.

**Comment by duydvu** [comment]
Yes, the primary column is currently hard-coded by `id`. I don't see any reason to keep it that way so it will be changed to adapt the table's primary key.
